flixel/text/FlxText.hx

```diff
@@ -804,33 +805,47 @@ class FlxText extends FlxSprite
 			
 			_matrix.identity();
 			
-			avoidSingleLineBlur();
-			
 			applyBorderStyle();
 			applyBorderTransparency();
 			applyFormats(_formatAdjusted, false);
 			
-			graphic.bitmap.draw(textField, _matrix);
+			drawTextFieldTo(graphic.bitmap);
 		}
 		
 		_regen = false;
 		resetFrame();
 	}
 	
-	private function avoidSingleLineBlur():Void
+	private function drawTextFieldTo(graphic:BitmapData):Void
 	{
 		#if flash
-		// If it's a single, centered line of text, we center it ourselves so it doesn't blur to hell
-		if (textField.numLines > 1 || alignment != FlxTextAlign.CENTER)
+		if (alignment == FlxTextAlign.CENTER)
+		{
+			// Prevents blurry lines.
+			var h:Int = 0;
+			var old_tx:Float = _matrix.tx;
+			var rect:Rectangle = new Rectangle();
```

Comment:
This is not a good idea, that's an allocation every time... `FlxText` inherits inherits a `_flashRect`, can we use that here?